internal/innosat/source.go

```diff
@@ -12,8 +12,22 @@ type SourcePacketHeaderType uint
 //
 //TC is the source package type for telecommand
 const (
-	TC SourcePacketHeaderType = 0
-	TM SourcePacketHeaderType = 1
+	TM SourcePacketHeaderType = 0
+	TC SourcePacketHeaderType = 1
+)
+
+//SourcePackageContinuationFlagType type for continuation groups
+type SourcePackageContinuationFlagType uint
+
+//SPCont Continuation packet
```

Comment:
I find I need an empty comment line for VSC not to just mangle the info as a single sentence with no spacing.